tests/parameter_handler/parameter_handler_read_json_05.cc

```diff
@@ -0,0 +1,51 @@
+// ------------------------------------------------------------------------
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+// Copyright (C) 2020 - 2024 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// Part of the source code is dual licensed under Apache-2.0 WITH
+// LLVM-exception OR LGPL-2.1-or-later. Detailed license information
+// governing the source code and code contributions can be found in
+// LICENSE.md and CONTRIBUTING.md at the top level directory of deal.II.
+//
+// ------------------------------------------------------------------------
+
+
+
+// Like parameter_handler_read_json_05 but output with
+// ParameterHandler::OutputStyle::KeepChanged.
+
+#include <deal.II/base/parameter_handler.h>
+
+#include "../tests.h"
+
+
+int
+main()
+{
+  initlog();
+
+  ParameterHandler prm;
+
+  double test_0 = 0;
+  double test_1 = 1;
```

Comment:
This one is skipped during printing, since value is the same as default.

Reply:
I think that's a very good / simple test.